### PR TITLE
Partner-Related Fixes and Upgrades

### PR DESCRIPTION
**Pull request by dpmesko**

### CODE REVIEW

File: fixtures/framework/README.md [hunk 1]
```diff
@@ -9,59 +9,77 @@ The following tools should already be installed
...
-OAUTH_ENABLED=true OAUTH_CLIENT_ID=<client-id> OAUTH_CLIENT_SECRET=<client-secret> OAUTH_DISCOVERY_URL=<discovery-url> OAUTH_JWT_USERNAME_PROPERTY=email HTTP_PORT=8080 NODE_HOST=localhost:8080 ./strato.sh --single
+cd <dir>/<dir>-server
+sudo yarn token-getter
 ```
 
-### Obtaining an ADMIN TOKEN
-
-This project requires an `ADMIN_TOKEN` env varibale for the deploy to work. This token can be obtained by using the `token-getter` utility packaged in `blockapps-rest`. This utility can be executed by running the following command:
-
+This command launches a small web server on port 80.
```
Comment (nikitamendelbaum) [inline fixtures/framework/README.md]: Can we give some more details here that the token-getter launches on same host (hostname:port) which is provided in `redirectUri` value in config file used?

File: fixtures/framework/README.md [hunk 2]
```diff
@@ -9,59 +9,77 @@ The following tools should already be installed
...
+cd <dir>/<dir>-server
+sudo yarn token-getter
 ```
 
-### Obtaining an ADMIN TOKEN
-
-This project requires an `ADMIN_TOKEN` env varibale for the deploy to work. This token can be obtained by using the `token-getter` utility packaged in `blockapps-rest`. This utility can be executed by running the following command:
-
+This command launches a small web server on port 80. 
+- In your brower, go to `http://localhost`
```
Comment (nikitamendelbaum) [inline fixtures/framework/README.md]: same, the URL will be the one from redirectUri, the resulting URL is displayed when token-getter is started

File: fixtures/framework/README.md [hunk 3]
```diff
@@ -9,59 +9,77 @@ The following tools should already be installed
...
-### Executing for development
+### Executing for Development
 
 #### Start nginx
 
 ```
-cd nginx-docker
-HOST_IP=$(ipconfig getifaddr en0) docker-compose up -d
+cd <dir>/nginx-docker
+HOST_IP=$(curl ifconfig.me) docker-compose up -d
```
Comment (nikitamendelbaum) [inline fixtures/framework/README.md]: this won't work properly if the host is behind the NAT. Not every machine in the network has the public IP address.
Let's use `172.17.0.1` for linux and `docker.for.mac.localhost` for mac.

File: index.js [hunk 4]
```diff
@@ -80,8 +86,8 @@ function printUsage(errMsg) {
 async function run(dir) {
   // TODO: Check for dependencies - yarn, create-react-app, docker
 
-  log(`Collecting configuration parameters...`);
-  await collectOauthDetails();
+  log(`Please enter the following configuration parameters:`);
```
Comment (nikitamendelbaum) [inline index.js]: noice, LOL 👍

File: index.js [hunk 5]
```diff
@@ -203,7 +201,7 @@ async function run(dir) {
 
   log(`\tSetting up UI`);
   log(`\t\tInitializing create-react-app...`);
-  spawn.sync("create-react-app", [uiDirectory]);
+  spawn.sync("npx", ["create-react-app", uiDirectory]);
```
Comment (nikitamendelbaum) [inline index.js]: not familiar with npx and what's it for. It's not the typo, is it? Just to confirm.

File: index.js [hunk 6]
```diff
@@ -284,7 +282,7 @@ async function run(dir) {
   fs.writeFileSync("README.md", readme);
 
   // TODO: Print usage instructions
-  log("Happy BUIDLing!");
+  log("Happy Building! :) ");
```
Comment (nikitamendelbaum) [inline index.js]: Haha, I can tell you joined the blockchain space after the 2017 hysteria era by just looking at that line change :)

File: index.js (hunk 5 above)
Comment (dpmesko) [reply index.js]: Spoke with Samrit about this one. `create-react-app` is deprecated, `npx create-react-app` is the new command

File: index.js (hunk 6 above)
Comment (dpmesko) [reply index.js]: is it some old blockchain joke I'm too young to understand? I thought it was a typo :)

Comment (dpmesko) [reply index.js]: just googled it...I feel dumb...

**Pull request merged**